### PR TITLE
Start servers for RUN_SELECTED_TESTS

### PR DESCRIPTION
**Pull request by bigmontz**

### CODE REVIEW

**Review by AndyHeap-NeoTech — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To run integration tests you need to:
     python3 -m unittest tests.neo4j.datatypes.TestDataTypes.testShouldEchoBack
     ```
   
-  * Alternatively, It's possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the tested in the `TEST_SELECTOR` environment variable. It's specialy useful during the development of new tests when it's used in combination with `run_all.py` enabling to run one specific test against all known drivers.
+  * Alternatively, It's possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the tested in the `TEST_SELECTOR` environment variable. It will start all dependencies needed (such as neo4j servers or tls server). It's specialy useful during the development of new tests when it's used in combination with `run_all.py` enabling to run one specific test against all known drivers.
```
Comment (robsdedude) [inline README.md]: Typos:

Alternatively, ~~It's~~*it's* possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the ~~tested~~ *tests* in the `TEST_SELECTOR` environment variable. ~~It~~ *This* will start all dependencies needed (such as neo4j ~servers~~ or tls ~~server~~ *servers*). It's ~~specialy~~ *especially* useful during the development of new tests *and* when ~~it's~~ used in combination with `run_all.py` enabling to run one specific test against all known drivers.

**Review by robsdedude — COMMENTED** [review]
Noted a few typos.

Furthermore, I think it would be pretty nice to be able to do the tests selection via a cmd flag and not only an env. variable (`TEST_SELECTOR`). Especially on windows where setting env vars is a pain, this would be very handy.

File: README.md (hunk 1 above)
Comment (bigmontz) [reply README.md]: Typos fixed.

**Comment by bigmontz** [comment]
> Furthermore, I think it would be pretty nice to be able to do the tests selection via a cmd flag and not only an env. variable (`TEST_SELECTOR`). Especially on windows where setting env vars is a pain, this would be very handy.

I will do it, i will make it get precedence over the env var.

File: README.md [hunk 2]
```diff
@@ -80,7 +80,9 @@ To run integration tests you need to:
     python3 -m unittest tests.neo4j.datatypes.TestDataTypes.testShouldEchoBack
     ```
   
-  * Alternatively, It's possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the tested in the `TEST_SELECTOR` environment variable. It's specialy useful during the development of new tests when it's used in combination with `run_all.py` enabling to run one specific test against all known drivers.
+  * Alternatively, it's possible to use the option `--tests RUN_SELECTED_TESTS` to build the driver backend and run the tested in the `TEST_SELECTOR` environment variable. This will start all dependencies needed (such as neo4j or tls servers). It's especially useful during the development of new tests when used in combination with `run_all.py` enabling to run one specific test against all known drivers.
+    
+    * The command-line param `--run-only-selected <test_selector>` works as shortcut of setting the `TEST_SELECTOR` environment variable and running the command with `--tests RUN_SELECTED_TESTS`.
```
Comment (robsdedude) [inline README.md]: This is very nit-picky and pleas feel free to ignore this part: I'd remove the blank line above that as the point belongs to the list above.

Another note: "~~works as~~ *is* a shortcut ~~of~~ *for*"

**Review by robsdedude — APPROVED** [review]
Two minor comments. Looks good to me. Nice feature \o/

File: README.md (hunk 2 above)
Comment (bigmontz) [reply README.md]: Done

**Pull request merged**